main.py

```diff
@@ -98,7 +98,6 @@ def generate_tc_config(version, enterprise, cluster, scheme, stress_test):
             ("3.5",    True,        False,    "bolt",   0),
             ("3.5",    True,        True,     "neo4j", 60),
             # not officially supported versions
-            ("4.0",    True,        False,    "neo4j",  0),
```

Comment:
this removes only 4.0, not 4.1 (which should be removed as well)

Reply:
I don't think so. 4.4 drivers were built to work with 4.1 servers and should still be tested against them.

Reply:
needs some more internal decision first for 4.1. Let's consider this resolved for now.